client/landing/gutenboarding/tsconfig.json

```diff
@@ -5,7 +5,7 @@
 		// Must be used with babel typescript
 		"isolatedModules": true,
 
-		"baseUrl": ".",
+		"baseUrl": "../..",
 		"paths": {
 			"*": [ "*", "../../*" ]
```

Comment:
I want to review this. I think the `baseUrl` above is necessary to allow calypso imports to resolve. If the paths here are relative to baseUrl —and I suspect they are—, they're definitely wrong:

```suggestion
			"*": [ "landing/gutenboarding/*", "*" ]
```